### ISSUE TITLE
Conda install

### ISSUE THREAD

**Issue by ppxasjsm** [michellab/BioSimSpace#81]
Following the instructions on using a conda isntall for BSS, I get the following errors:
`conda install -c conda-forge -c omnia -c michellab biosimspace`
Collecting package metadata: done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - biosimspace

Current channels:

  - https://conda.anaconda.org/conda-forge/linux-64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://conda.anaconda.org/omnia/linux-64
  - https://conda.anaconda.org/omnia/noarch
  - https://conda.anaconda.org/michellab/linux-64
  - https://conda.anaconda.org/michellab/noarch
  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/free/linux-64
  - https://repo.anaconda.com/pkgs/free/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.

**Comment by lohedges** [comment]
This is because we still don't have a 2019.1.0 tagged release, so there are no Conda packages for Sire or BioSimSpace with the `main` label on our Conda channel. At the moment there are only development packages, which must be installed using, e.g.:

```sh
conda install -c conda-forge -c omnia -c michellab/label/dev biosimspace
```

The documentation was written on the assumption that we had an imminent release.

**Comment by chryswoods** [comment]
Please feel free to tag the release. The code looks like it is working. You just need to update the version number and then merge into master.

**Comment by lohedges** [comment]
I'm happy to do this, I was just waiting for the go-ahead from the Cresset folks regarding their testing of the updated AmberPrm parser. I'd like to tag Sire then BioSimSpace, since the BioSimSpace Conda package will need the tagged Sire release as a dependency.

I think you already updated the version numbers back in January (I see 2019.1.0 in [corelib/CMakeLists.txt](https://github.com/michellab/Sire/blob/devel/corelib/CMakeLists.txt)) so it should just be a case of tagging the latest commit in devel, then merging to master. If I re-run the Azure Pipeline for devel, it should then automatically generate the release Conda packages.

**Comment by chryswoods** [comment]
Yes - sounds like a good plan. Yes, I did update the version number in January. Please update to 2019.1.1 so that we can be distinct.

**Comment by lohedges** [comment]
Although you updated the version number back in January, there isn't a 2019.1.0 tag for Sire and a release doesn't appear [here](https://github.com/michellab/Sire/releases). Do you want to skip this version entirely and move to 2019.1.1? (Both for Sire and BioSimSpace.)

**Comment by chryswoods** [comment]
Closing as you have already released 2019.1.0 (which is fine)
